src/lib/sw_engine/tvgSwRenderer.cpp

```diff
@@ -324,6 +324,7 @@ bool SwRenderer::renderShape(RenderData data)
         if (a > 0) rasterSolidShape(surface, &task->shape, r, g, b, a);
     }
 
+    a = 0;
```

Comment:
If you want to determine whether or not a stroke is created, I think it is good to check the return value of strokeColor for the readability of the code.

How about this
```cpp
if (task->sdata->strokeColor(&r, &g, &b, &a) == Result::Success)
```

Reply:
yours looks more elegant.
are we ok with an extra if statement?

Reply:
changed according to your recommendation